setup.py

```diff
@@ -36,7 +36,9 @@ def get_version(rel_path):
         'schedule',
         'prometheus-client',
         'PyYAML',
-        'requests'
+        'requests',
+        'opentelemetry-sdk',
```

Comment:
I'll remove this from the actual release as pollect is shipped with as little dependencies as possible, and the most common use case is still prometheus atm.